spectrum/autism_prevalence_map/templates/autism_prevalence_map/map.html

```diff
@@ -14,52 +14,43 @@
 {% block body_block %}
 <!-- Main Container
 –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-<div class="container-fluid">    
-    <div class="row">
-        <div class="col no-padding-col">
-            <!-- Map div
-            –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-            <div id="map"></div>
-        </div>
-    </div>
-    <div class="row">
-        <div class="col no-padding-col">
-            <!-- Timeline div
-            –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-            <div class="timeline-wrapper container-fluid">
-                <div class="row">
-                    <div id="timeline-toggle" class='timeline-toggle col-xl-3'>
-                        <h2>Timeline of studies</h2>
-                        <span class="timeline-switch">
-
-                            <input type="checkbox" id="timeline-switch" name="timeline-switch" {% if timeline_type == "studied" %}checked{% endif %}>
-
-                            <label for="timeline-switch">
-                                {% if timeline_type == "published" %}
-                                    <strong>Year published</strong>
-                                {% else %}
-                                    Year published
-                                {% endif %}
-                            </label>
-                            <span class="timeline-switch-knob"></span>
-                        </span>
-                        <span>
-                            {% if timeline_type == "studied" %}
-                                <strong>Year(s) studied</strong>
-                            {% else %}
-                                Year(s) studied
-                            {% endif %}
-                        </span>
-                        {% if last_updated_on != "" %}
-                            <div>
-                                Data last updated on {{last_updated_on}}
-                            </div>
+<div class='mt-32-75 h-screen'>    
+    <!-- Map div
+    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+    <div id="map" class=""></div>
```

Comment:
use single quote here and below

Reply:
I figure whoever rewrites this can take care of that; I didn't actually add any of these lines and they may not be part of the final code